core/server/data/migrations/versions/2.18/1-restore-settings-from-backup.js

```diff
@@ -22,6 +22,11 @@ module.exports.up = (options) => {
     }, options);
 
     return fs.readdir(dataPath).then(function (files) {
+        return files
+    }).catch(function () {
```

Comment:
why not global catch below?

Reply:
Because it will catch errors with writing to the db

Reply:
Do we want to catch this?

Reply:
I'd not catch each error.
Just at the end and one general message.

Reply:
no

Reply:
If there is something wrong with his db in general, he will for sure notice when he starts his blog

Reply:
Just my 2 cents. Can discuss in slack if you want :)

Reply:
Let's keep like it is 👍